### DISCUSSION TITLE
Bibliographic resources and discussion of them

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1241] [users: ideas for new features]
Transfered here from the wiki article "ideas for analysis":

We have a ton of scanned articles but they live in different places and CMK has a tendency to clean out his dropbox and deprive me of things. I want to consider having scanned resources in the File Repository. This is especially important because, for instance, Brown and Swansea have different resources and I can't get everything or share everything unless we have a communal place to actually deposit scans. 

- But then it would also be incredibly helpful to have a way of having team members able to annotate or write descriptions of articles they have read, too, so that linked to the actual articles are fora for discussing them during research and publication. So for instance MRM is reading a bunch of geological stuff. I'm not going to dive as in depth into that material as she will, but I want her notes even now and to comment on directions to take thoughts. (She's also done a ton of the scanning of stuff.) 

- A further related thought that is less likely to come to fruition is linking to bibliographic software in some way that keeps an Uronarti bibliography in a way that can be exported for publications. But that I have never managed to do even in less sprawling research and publication situations so I am not convinced I would use that functionality even if we were able to implement it.

**Comment by urapadmin** [comment]
I want to add that references to literature should not only link to the source on either the internet or the file repository or both but also should be displayed in an accepted style. I always wanted to write a simple styling thingy that takes in data and spits out the correct line for a bibliography. I know, there is software for that, but most of what I have seen is over designed and tries too much. 

What styles would be most important to begin with (my idea is that one could either configure a style for a project or even different styles and the user selects the most useful depending on preference). And yet, I need to start with something useful. so what would we need most? MLA?

**Reply by lbestock** [reply]
[Chicago author-date!](https://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-2.html) Oh wow and I volunteer to test this one out lickety split because with my current crazy schedule I am reading stuff in lightspeed for classes and not keeping track of it and making a mess and then putting up improperly formatted citations in Canvas. Which, since I ask my students to cite things properly, is not ideal.

**Reply by urapadmin** [reply]
ja, you read too much into this. Won't help you with a lack of discipline. But I got my answer: Chicago author-date. 
Any others? @luizaogs?

**Reply by urapadmin** [reply]
added as a candidate for a lab project to #1239

**Reply by luizaogs** [reply]
I use Chicago notes and bibliography most often (https://www.chicagomanualofstyle.org/tools_citationguide/citation-guide-1.html). (Really I combine the two and make my own, but that doesn't help you.)

**Comment by urapadmin** [comment]
> But then it would also be incredibly helpful to have a way of having team members 
>able to annotate or write descriptions of articles they have read, too, so that linked to the actual articles are fora for discussing 
>them during research and publication. So for instance MRM is reading a bunch of geological stuff. I'm not going to dive as in 
>depth into that material as she will, but I want her notes even now and to comment on directions to take thoughts. 
>(She's also done a ton of the scanning of stuff.)
> 
That does seem to take it a bit too far for my taste in our software, no? At least annotating the articles themselves does. That would need a viewer for an article first and then tools to mark and annotate. That is really very specialized software. 

Storing excerpts or descripions together with links to the article and bibliography and all that connected to the archeological record I can imagine.
